Formula/man-db.rb

```diff
@@ -0,0 +1,45 @@
+class ManDb < Formula
+  desc "Modern, featureful implementation of the Unix man page system"
+  homepage "https://nongnu.org/man-db/"
+  url "https://download.savannah.nongnu.org/releases/man-db/man-db-2.8.5.tar.xz"
+  sha256 "b64d52747534f1fe873b2876eb7f01319985309d5d7da319d2bc52ba1e73f6c1"
+
+  depends_on "libpipeline"
+
+  def install
+    system "./configure", "--prefix=#{prefix}",
+                          "--with-systemdtmpfilesdir=no",
+                          "--with-systemdsystemunitdir=no",
+                          "--disable-cache-owner",
```

Comment:
Why are these options necessary, if non-default?

Reply:
See ylluminarious/homebrew-man-db@00f49e4 and ylluminarious/homebrew-man-db#1.

This PR isn't my work.

Reply:
`--with-systemdtmpfilesdir=no` and `with-systemdsystemunitdir=no` are workarounds for Linuxy defaults in man-db.  I've just [fixed those upstream](https://git.savannah.gnu.org/cgit/man-db.git/commit/?id=b74c839eaa5000a18d1c396e995eca85b0e9464b), so they won't be necessary with man-db >= 2.8.6 once I release that.  (Maybe add a comment to that effect if possible.)

`--disable-cache-owner` and `--disable-setuid` disable the default behaviour of having a cache of formatted pages shared between users via a special `man` user, in line with the Homebrew policy of not requiring root.